Segs/Segs
# SEGSAdmin Field Validation Update

Closes #437 
Closes #433 

**This update to SEGSAdmin introduces the following**

- Validation on the following fields

server name - 32
max players - counter, max 200
max char slots - counter, 8
map location - 32	
account username - 14
account password - 14

- Add validation on blank fields on saving Settings. If blank, user is shown the problem fields by use of highlight and a warning message
- Fixes a crash on opening 'Settings & Logging' dialog if settings.cfg file does not exist yet
- Add QComboBox for selection of DB Drivers
- player_fade_in now utilises a slider and max value of 1000
- player_fade_in will save as a float ensuring .0 is appended to the saved value


**Update 06/07**

Add spin boxes for below port fields, now validate for a value between 1024 and 65536:

char db port
acc db port
auth port
map list port 
map loc port
game list port
game loc port

Add a check on username and password that they are 'Latin' (toLatin1()) and do not contain invalid UTF-8 characters
